### PR TITLE
integrated the requirements and BOM in the readme and added project p…

### PR DESCRIPTION
**Pull request by wassfila**
suggestion to have a single page readme as it's hard to navigate between pages in github repo
added some suggestions for the project progress and maturity levels
please feel free to review and suggest changes
all contributors would be invited to review this PR as per the suggested guidelines for key files

### CODE REVIEW

File: SuperPower-uC/README.md [hunk 1]
```diff
@@ -33,3 +44,61 @@ Examples:
...
+  * 3 ) functional
+  * 4 ) stable and bug free
+* The maturity level is decided by the Team during a meeting
+
+# Requirements
+With the help of a lot participants we collected a list of requirements for this project.
+
+## Filtered Requirements
+[Requirements ESP32 Boards shared spreadsheet](https://docs.google.com/spreadsheets/d/1Awvs3CumJ9qQSrG_uEvt4s6nZwliIjnMdfPibDWA3kU/edit#gid=1880015614&fvid=1986096210)
+
```
Comment (Vincent-Stragier) [inline SuperPower-uC/README.md:73]: It could be good to add a list with the possible categories and their description. (Not mandatory)

**Review by Tobi3566 — COMMENTED** [review]
wouldn't the BOM be original/low cost specific? rest looks good

**Comment by Informaticore** [comment]
> wouldn't the BOM be original/low cost specific? rest looks good

yes thats why there are sections for each - but I also thing a new page for each variant is better. but we can change that later as well

**Review by Tobi3566 — APPROVED** [review]

File: SuperPower-uC/README.md (hunk 1 above)
Comment (wassfila) [reply SuperPower-uC/README.md:73]: I agree, I think I will add that in a next version, I would also like to add a status for each requirement and maybe a maturity level such as not only accepted / rejected, but also implemented / tested and so on. In this PR I only performed a reformatting.

**Comment by wassfila** [comment]
> wouldn't the BOM be original/low cost specific? rest looks good

It is, the BOM is a header of lower level under each variant, I could renaming to orginal variant BOM to be clearer, and when the low cost BOM would have a BOM added it would be a location where it's possible to compare BOMs of variants without having to open multiple pages.
I also only added the BOM because it's not big, but once we have a bigger doc for each variant as @Informaticore said, we can move them in separate pages.

**Comment by wassfila** [comment]
I add a comment here, in the future I'll think about adding clearer rules for anything we copy from internet and not created by the committer, requiring a reference or a link in readme so that the licensing or authorization can be checked.

**Comment by wassfila** [comment]
In order to avoid changing back and forth, let me know if you all want each BOM and sub variant description in separate readme file ?
Or maybe we keep the original only on the main readme file ?
So you can vote thumbs up or thumbs down if you agree / disagree with the following options

**Comment by wassfila** [comment]
All BOMS of all variants in the main uC readme

**Comment by wassfila** [comment]
All BOMS separate in each sub directory readme

**Comment by wassfila** [comment]
Original version BOM in the main readme and all variants BOM in separate files

**Review by Informaticore — APPROVED** [review]

**Review by MantaRayDeeJay — APPROVED** [review]
I don't have any request to add. If we need some changes, we will do next time.

**Review by dr-stefan-krank — APPROVED** [review]

**Pull request merged**